sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -291,6 +291,49 @@ certificate will be added with the alias of `mycert`.
 2. Certificates coming from Azure Key Vault take precedence over 
 side-loaded certificates.
 
+### Enable mutual TLS on the server side
+
+Only some minor changes need to be done to the server side SSL example 
+mentioned above.
+
+The following additional application.properties need to be added:
+
+```
+server.ssl.client-auth=need
+server.ssl.trust-store-type=AzureKeyVault
+```
```

Comment:
Could you please change all `.properties` to `.yml`?
1.  Rename `application.properties` to `application.yml`
2. Update content:
```
server:
  ssl:
    client-auth: need
    trust-store-type: AzureKeyVault
```